backend/danswer/direct_qa/__init__.py

```diff
@@ -43,24 +47,52 @@ def check_model_api_key_is_valid(model_api_key: str) -> bool:
 
 def get_default_backend_qa_model(
     internal_model: str = INTERNAL_MODEL_VERSION,
-    api_key: str | None = None,
+    endpoint: str | None = GEN_AI_ENDPOINT,
+    model_host_type: str | None = GEN_AI_HOST_TYPE,
+    api_key: str | None = get_gen_ai_api_key(),
```

Comment:
`get_gen_ai_api_key` will only be called once on startup here. That's not intended right? 

If not, we should do something like: "give `api_key` a default value of none, and then add a `if api_key is None` in the body of the function to give it a default value"

Reply:
mmm ya that's better